gitlabform/gitlab/groups.py

```diff
@@ -122,11 +122,6 @@ def put_group_settings(self, project_and_group_name, group_settings):
             "groups/%s", project_and_group_name, "PUT", group_settings
         )
 
-    def get_group_shared_with(self, group):
-        group = self.get_group_case_insensitive(group)
-
-        return group["shared_with_groups"]
```

Comment:
Removed because this attribute can be accessed directly from a python-gitlab `Group` instance.